src/Microsoft.PowerShell.ConsoleHost/host/msh/UpdatesNotification.cs

```diff
@@ -190,8 +209,8 @@ internal static async Task CheckForUpdates()
                     // Do the real update check:
                     //  - Send HTTP request to query for the new release/pre-release;
                     //  - If there is a valid new release that should be reported to the user,
-                    //    create the file `update_<tag>_<publish-date>` when no `update` file exists,
-                    //    or rename the existing file to `update_<new-version>_<new-publish-date>`.
+                    //    create the file `<NotificationType>Update_<tag>_<publish-date>` when no `update` file exists,
+                    //    or rename the existing file to `<NotificationType>Update_<new-version>_<new-publish-date>`.
```

Comment:
It seems it is `Update<NotificationType>_<tag>_<publish-date>`

Reply:
Good catch. Yes, the name was updated but I didn't change the comments.